### ISSUE TITLE
[Proposal] Be able to make 'like' query on fields

### ISSUE THREAD

**Issue by LuizVisnadi** [lorisleiva/laravel-search-string#30]
Hello,

I would like to propose a new feature to this package.

Currently, when we need to make a 'like' type search, we need to define the model attributes as "searchable".
However, we cannot specify if we only want to make a 'like' search on only one attribute.

Let me explain with an example, I have a model User where I've defined the attributes 'firstname' and 'lastname' as _searchable_.
Most of the time, my search concerns both attributes, but sometimes I need to make a 'like' search on only one of them.

With this need in mind, I would like to propose an syntax to be able to make a 'like' type search without defining the model attribute as _searchable_. I've thought of something like this: **'field_:~_value'** and **'field_=~_value'**.

**tl:dr :** I propose the syntax (**'field_:~_value'**, **'field_=~_value'**) to be able to make "_where field like '%value%'_" query.

**Comment by lorisleiva** [comment]
Hi there 👋 

I'm afraid this has already been proposed and I still believe it is not the way to go for this package. https://github.com/lorisleiva/laravel-search-string/pull/12#issuecomment-593102787

However, I wouldn't be opposed for a PR that adds an option on the column to alter the behaviour of the equal operator and transform it into a like query. Something like:

```php
protected $searchStringColumns = [
    'firstname' => [
        'use_like_operator' => 'both', // Supports: 'left' for '%text', 'right' for 'text%' and 'both' for '%text%'
        // ...
    ],
    // ...
];
```
